blocks/init/src/Blocks/components/share/share.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Template for the Share Component view.
+ *
+ * @package EightshiftBoilerplate
+ */
+
+use EightshiftBoilerplateVendor\EightshiftLibs\Helpers\Components;
+
+$manifest = Components::getManifest(__DIR__);
+
+$postUrl = \get_the_permalink();
```

Comment:
this options should be passed as a props an not defined fixed here. 
If for some reason you want to use this component outside of posts this wont work. Please add all of the options via props, you can defined this values as fallback if nothing is passed